src/main/java/net/fabricmc/loader/api/metadata/ModMetadata.java

```diff
@@ -63,6 +64,7 @@
 
 	boolean containsCustomValue(String key);
 	CustomValue getCustomValue(String key);
+	Map<String, CustomValue> getCustomValues();
```

Comment:
Mind quickly document if this map is unmodifiable?

Reply:
Yes it is unmodifable, (See the CustomValueContainer).

I'll add docs for it if you deem it nessecary

Reply:
None of the other methods are documented at the moment, and I feel like it might make more sense to document the (un)modifiability of every method in the interface javadoc rather than this method in particular.

Reply:
Hmm. Yeah that makes sense, I'd probably open up a separate PR to document those other methods in that case.